source/configuration_manual/shared_mailboxes/cluster_setup.rst

```diff
@@ -35,6 +35,13 @@ to access shared mailboxes of users which live on other backends via the IMAP
 protocol. For this purpose shared mailboxes in a cluster must be accessed via
 another namespace which has ``location`` configured to use ``imapc``.
 
+It is recommended to disable shared namespaces for master users so you should
+add ``userdb_namespace/shared/disabled=yes`` to your master passdb as
+:ref:`extra field <authentication-password_database_extra_fields>`. Depending on
+the authentication configuration it can be necessary to also override the ``acl_user``
```

Comment:
I'm not sure if there's a difference, but "it may be necessary" sounds more correct to me..